### PR TITLE
Clarified support for journald per BZ

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1526262

### CODE REVIEW

**Comment by mburke5678** [comment]
I added the wording from the release notes to the _Aggregating Container Logs_ topic (removing the existing wording from the *NOTE*). 
I removed the _Aggregated logging is only supported using the journald driver in Docker._ wording from the _Host Preparation_ page, as the section seems focused on `json-file` and log size, not `journald`. 
Are there concerns with `journald` log size that we want to add somewhere?  
How does a user changes between `json-file` and `journald`?

File: install_config/install/advanced_install.adoc [hunk 1]
```diff
@@ -486,10 +486,16 @@ which may cause invalid configurations. Example usage:
...
+a|This variable configures additional `docker` options within
 *_/etc/sysconfig/docker_*, such as options used in
-xref:../../install_config/install/host_preparation.adoc#managing-docker-container-logs[Managing Container Logs]. Example usage: *"--log-driver json-file --log-opt max-size=1M
---log-opt max-file=3"*. Do not use when
+xref:../../install_config/install/host_preparation.adoc#managing-docker-container-logs[Managing Container Logs]. 
+Use `json-file` or `journald`. The default is `json-file`. 
+Example usage: 
+----
+"--log-driver json-file --log-opt max-size=1M --log-opt max-file=3"
+"--log-driver journald --log-opt max-size=1M --log-opt max-file=3"
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply install_config/install/advanced_install.adoc]: @anpingli If the user wants to change these settings after install, could he use the *_/etc/docker/daemon.json_*? 
Are these fields needed in the *_/etc/ansible/hosts_*?

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -42,15 +42,24 @@ administrators can view all logs, but application developers can only view logs
...
 to manage container logs and prevent filling node disks.
 
-Aggregated logging is only supported using the `journald` driver in Docker. See
-xref:fluentd-upgrade-source[Updating Fluentd's Log Source After a Docker Log
-Driver Update] for more information.
+
+[IMPORTANT]
+====
+If Docker log-driver is set to journald, there is no log rate throttling with the `journald` driver. 
+As a result, there is always a risk for denial-of-service attacks from rogue containers.
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply install_config/aggregate_logging.adoc]: @anpingli Added:
_When using the `json-file` driver, ensure that your Docker version is Docker version *docker-1.12.6-55.gitc4618fb.el7_4 now* or greater._

Is this correct?

**Comment by mburke5678** [comment]
@anpingli PTAL

File: install_config/install/advanced_install.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (anpingli) [reply install_config/install/advanced_install.adoc]: Yes, They can change the values  in  /etc/docker/daemon.json directly.

**Review by anpingli — APPROVED** [review]

File: install_config/install/advanced_install.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (anpingli) [reply install_config/install/advanced_install.adoc]: change "--log-driver journald --log-opt max-size=1M --log-opt max-file=3"  to  "--log-driver journald ". The other part looks good.

**Comment by mburke5678** [comment]
@anpingli  Do you know what OpenShift versions this issues applies to? How far back do we support json-file and journald?

File: install_config/aggregate_logging.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (mburke5678) [reply install_config/aggregate_logging.adoc]: @anpingli PTAL ^^

**Comment by mburke5678** [comment]
@anpingli Do you know what OpenShift versions this issues applies to? How far back do we support json-file and journald?

**Comment by anpingli** [comment]
@mburke5678, Openshift  support both journald/json file driver always.    In v3.4, the default driver is json-file if you deploy openshift using openshift-ansile.   In v3.5-v3.7, For Bug526262,   The default driver is changed to journald.  In v3.9,  we had changed backup to json driver.

File: install_config/aggregate_logging.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (anpingli) [reply install_config/aggregate_logging.adoc]: Yes

[the remark this answers is not in the document]
Comment (anpingli) [reply install_config/aggregate_logging.adoc]: The latest docker is docker-1.12.7.  No such issue. I  recall my suggestion.

**Comment by mburke5678** [comment]
@anpingli The 3.7 release notes say that json-file is the default. But, you suggest that journald is the default. (I tested in a 3.5 Ravello system and I get journald).

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL. Just need to close on the default logging driver by OSP version. But, a peer review can be done on the text.

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL.

**Comment by mburke5678** [comment]
Note to self: Follow up in 3.9; change default to json-file

File: install_config/aggregate_logging.adoc [hunk 3]
```diff
@@ -1654,6 +1662,58 @@ $ curl --key /etc/elasticsearch/secret/admin-key \
 ----
 ====
 
+[[fluentd-update-source]]
+== Changing the Aggregated Logging Driver
+
+By default, aggregated logging uses the `journald` log driver 
+xref:../install_config/install/advanced_install.adoc#configuring-host-variables[unless `json-file` was specified during installation]. 
+You can change the log driver between `journald` and `json-file` as needed.
```
Comment (ahardin-rh) [inline install_config/aggregate_logging.adoc]: Is "You can change..." supposed to be on a new line?

File: install_config/aggregate_logging.adoc [hunk 4]
```diff
@@ -1654,6 +1662,58 @@ $ curl --key /etc/elasticsearch/secret/admin-key \
...
+
+[IMPORTANT]
+====
+When using the `json-file` driver, ensure that your Docker version is Docker version *docker-1.12.6-55.gitc4618fb.el7_4 now*
+or greater.
```
Comment (ahardin-rh) [inline install_config/aggregate_logging.adoc]: s/greater/later

**Comment by ahardin-rh** [comment]
@mburke5678 Just a few nits from me. Otherwise, looks good! :star2:

**Comment by mburke5678** [comment]
Fixed release notes that was missed when working on this PR

See https://bugzilla.redhat.com/show_bug.cgi?id=1526262#c16

**Pull request merged**